x-pack/legacy/plugins/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -56,29 +65,42 @@ export const Page: FC = () => {
   // Call useRefreshInterval() after the subscription above is set up.
   useRefreshInterval(setBlockRefresh);
 
+  const [isSearchSelectionVisible, setIsSearchSelectionVisible] = useState(false);
+  const [savedObjectId, setSavedObjectId] = useState<string | null>(null);
+
+  if (savedObjectId !== null) {
+    return <RedirectToCreateTransform savedObjectId={savedObjectId} />;
+  }
+
+  const onCloseModal = () => setIsSearchSelectionVisible(false);
+  const onOpenModal = () => setIsSearchSelectionVisible(true);
+
+  const fakeVisType = {
```

Comment:
Could do with a comment as to what the purpose of this is `fakeVisType` is for.

Reply:
In 89d3398 I got rid of `fakeVisType` and implemented it the way Dima did it for the reactified page in the ML plugin: I copied the `SearchSelection` component from visualize code and adapted it so we don't have to hack our way into visualize's code.